share/completions/hostnamectl.fish

```diff
@@ -1,18 +1,21 @@
-set -l commands status set-hostname set-icon-name set-chassis set-deployment set-location
```

Comment:
Optional fix: if `--version` or option with the same semantics is available please use it to decide what completions to provide.